src/midi_Namespace.h

```diff
@@ -27,7 +27,7 @@
 
 #pragma once
 
-#define MIDI_NAMESPACE                  midi
+#define MIDI_NAMESPACE                  midi_v440
```

Comment:
Changing the namespace here will break existing code that references it directly (eg: `midi::ControlChange`)

Reply:
Fixed